content/04.study.md

```diff
@@ -473,3 +473,20 @@ However, because this model was not subsequently tested on ground-truth datasets
 
 Method development for interpreting new types of sequencing data has historically taken two steps: first, easily implemented hard cutoffs that prioritize specificity over sensitivity, then expert development of probabilistic models with hand-developed inputs [@tag:Torracinta2016_sim].
 We anticipate that these steps will be replaced by deep learning, which will infer features simply by its ability to optimize a complex model against data.
+
+
+
+### Neuroscience
+
+Artificial neural networks were originally conceived as a model for computation in the brain.
+The deep artificial neural networks of today have evolved to become a workhorse in computer science, with broad applications across many fields outside of neuroscience.
```

Comment:
I think the broad applications will be evident by this point in the review.  How about collapsing this sentence and the next into something like

> Although deep neural networks have evolved to become a workhorse across many fields, there is still a strong connection between deep networks and the study of the brain.